src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Payload.scala

```diff
@@ -42,6 +42,19 @@ class Payload {
   }
 
   /**
+   * Overloaded add function for Option. Don't modify payload for None
+   *
+   * @param name parameter name
+   * @param value optional parameter value
+   */
+  def add(name: String, value: Option[String]): Unit = {
```

Comment:
I'm using it everywhere because procedure syntax will be deprecated in future versions of Scala.